index.md

```diff
@@ -37,3 +37,4 @@ by our [code of conduct](http://software-carpentry.org/conduct.html).
 *   [Reference Guide](reference.html)
 *   [Next Steps](discussion.html)
 *   [Instructor's Guide](instructors.html)
```

Comment:
Nit picking:
I suggest to use "Trainer's Guide" to consistently and stubbornly propagate the terminology where instructors are those teaching workshops and trainers are those teaching instructors (in many organisations our "Instructor Training is called "Train the Trainer" and inevitably this will lead to confusion).